axon-server-connector/src/main/java/org/axonframework/axonserver/connector/util/ExecutorServiceBuilder.java

```diff
@@ -39,21 +39,18 @@
 public interface ExecutorServiceBuilder extends
         BiFunction<AxonServerConfiguration, BlockingQueue<Runnable>, ExecutorService> {
 
-    long THREAD_KEEP_ALIVE_TIME = 100L;
-
     /**
      * Create a default ExecutorServiceBuilder used to create a {@link ThreadPoolExecutor} for processing incoming
-     * commands. Uses the {@link AxonServerConfiguration#getCommandThreads()} as the core and maximum pool size, a
-     * keep-alive time of {@code 100ms}, the given {@link BlockingQueue} as the {@code workQueue} and an {@link
-     * AxonThreadFactory}.
+     * commands. Uses the {@link AxonServerConfiguration#getCommandThreads()} as the core and maximum pool size,
+     * the given {@link BlockingQueue} as the {@code workQueue} and an {@link AxonThreadFactory}.
      *
      * @return a default ExecutorServiceBuilder to create an executor for processing commands
      */
     static ExecutorServiceBuilder defaultCommandExecutorServiceBuilder() {
         return (configuration, commandProcessQueue) -> new ThreadPoolExecutor(
                 configuration.getCommandThreads(),
                 configuration.getCommandThreads(),
-                THREAD_KEEP_ALIVE_TIME,
+                0L,
```

Comment:
I trust the change, but I do want to understand why you made the change. So, care to elaborate?

Reply:
I assume because of that:
https://github.com/AxonFramework/AxonFramework/pull/3260#discussion_r1940839626

@CodeDrivenMitch maybe it's good to comment it shortly?

Reply:
The thread keepalive time is the time the pool will keep unused threads alive before killing it.
As this configuration has the same minimum and maximum threads, it will never kill any thread and keep them all alive.
Based on a comment from @abuijze on my Javadoc. 
As there is no way to kill threads, I thought to remove the keepalive time variable completely as it might mislead.
We can also choose to keep it, but at least remove it from the javadoc.

Reply:
Alright, that's sufficient for me. Thanks for the explanation!